manifests/crd-alerting-rule.yml

```diff
@@ -11,3 +31,24 @@ spec:
     singular: alertingrule
     plural: alertingrules
   scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            group:
+              type: string
+            alert:
+              type: string
+            expr:
+              type: string
+            for:
+              type: string
+              pattern: '^(\d+h)?(\d+m)?(\d+s)?(\d+ms)?(\d+us)?(\d+ns)?$'
+            annotations:
+              type: object
+            labels:
+              type: object
+          required:
```

Comment:
Shouldn't `group` be mandatory?

Reply:
The code of the `prometheus-launcher` assigns `default` if there is no group, so technically it isn't mandatory. But in the Prometheus world all alerting roles should go inside a group, so you are right, it should be mandatory, I will change that.